indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/autoscaler/LagBasedAutoScaler.java

```diff
@@ -37,6 +40,8 @@
 
 public class LagBasedAutoScaler implements SupervisorTaskAutoScaler
 {
+  public static final String SKIP_REASON_DIMENSION = "skipReason";
```

Comment:
`skipReason` seems a little ambiguous.
```suggestion
  public static final String SKIP_REASON_DIMENSION = "scalingSkipReason";
```